src/types/response/futures.types.ts

```diff
@@ -422,6 +424,10 @@ export interface FuturesPosition {
   maintainMargin: number; // Maintenance margin rate
   userId: number; // userId
   riskLimitLevel: number; // Risk Limit Level
+  marginMode: 'ISOLATED' | 'CROSS'; // Margin mode
+  positionSide: string;
```

Comment:
Is there an enum for the possible values here? Would be good to also have a union type here